std_msgs/Header header
bool left_validity
bool right_validity
geometry_msgs/Point left_xyz
geometry_msgs/Point right_xyz